### ISSUE TITLE
Uncaught TypeError: Cannot read property 'relativePath' of undefined

### ISSUE THREAD

**Issue by nfantone** [benjaminRomano/gulp-manager#1]
[Enter steps to reproduce below:]
1. Open Gulp pane
2. Click on the _refresh_ icon.

**Atom Version**: 1.0.5
**System**: Linux Mint
**Thrown From**: [gulp-manager](https://github.com/benjaminRomano/gulp-manager) package, v0.2.3
### Stack Trace

Uncaught TypeError: Cannot read property 'relativePath' of undefined

```
At /home/nfantone/.atom/packages/gulp-manager/lib/views/output-view.coffee:50

TypeError: Cannot read property 'relativePath' of undefined
  at OutputView.addGulpTasks (/home/nfantone/.atom/packages/gulp-manager/lib/views/output-view.coffee:50:50)
  at OutputView.refresh (/home/nfantone/.atom/packages/gulp-manager/lib/views/output-view.coffee:121:8)
  at GulpPaneView.refresh (/home/nfantone/.atom/packages/gulp-manager/lib/views/gulp-pane.coffee:37:17)
  at DockPaneManager.refreshPane (/home/nfantone/.atom/packages/bottom-dock/lib/views/dock-pane-manager.coffee:38:14)
  at BottomDock.refreshPane (/home/nfantone/.atom/packages/bottom-dock/lib/views/bottom-dock.coffee:41:22)
  at Emitter.module.exports.Emitter.emit (/usr/share/atom/resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at TabManager.refreshClicked (/home/nfantone/.atom/packages/bottom-dock/lib/views/tab-manager.coffee:52:16)
  at TabManager.refreshClicked (/home/nfantone/.atom/packages/bottom-dock/lib/views/tab-manager.coffee:1:1)
  at HTMLSpanElement.<anonymous> (/usr/share/atom/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:220:36)
  at HTMLSpanElement.jQuery.event.dispatch (/usr/share/atom/resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
  at HTMLSpanElement.elemData.handle (/usr/share/atom/resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)

```
### Commands

```
  4x -7:37.9.0 core:backspace (atom-text-editor.editor.mini.is-focused)
  2x -7:35 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:39.3.0 bottom-dock:toggle (atom-text-editor.editor.is-focused)
     -0:33.8.0 gulp-manager:add (atom-text-editor.editor.is-focused)
     -0:29.5.0 bottom-dock:toggle (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "themes": [
      "one-dark-ui",
      "seti-syntax"
    ],
    "projectHome": "/home/nfantone/dev/js"
  }
}
```
### Installed Packages

``` coffee
# User
atom-jade, v0.3.0
autocomplete-paths, v1.0.2
bottom-dock, v0.2.0
build, v0.43.0
color-picker, v2.0.11
docs-snippets, v0.8.0
editorconfig, v1.0.1
file-icons, v1.6.2
gulp-manager, v0.2.3
linter, v1.3.2
linter-eslint, v3.0.2
linter-jsonlint, v0.1.4
seti-syntax, v0.4.0
terminal-status, v1.6.8

# Dev
No dev packages
```

**Comment by benjaminRomano** [comment]
Thanks for filing the bug. I found the problem, and I should have it fixed shortly.

**Comment by benjaminRomano** [comment]
Should be fixed by c7e4511e02af27dcc95e9ecf0583f6384e242627. Can you verify that refreshing works now? Thanks.

**Comment by nfantone** [comment]
It works now. Great! 

Thank you.
